docs/NNNN-direct-clean-path-layout.md

```diff
@@ -0,0 +1,456 @@
+# OCFL Community Extension NNNN: Direct Clean Path Layout
+
+* **Extension Name:** NNNN-direct-clean-path-layout
+* **Authors:** Jürgen Enge, Basel
```

Comment:
Trivial change - I think it should be `Jürgen Enge (Basel)` to avoid the misreading of two authors, one called `Basel`

Reply:
Thanks, done